tasks/check_requirements.yml

```diff
@@ -3,7 +3,3 @@
   - name: Check if OS is Debian-based (we do not support others)
     debug: msg="Check OS family"
     failed_when: ansible_os_family != "Debian"
-
-  - name: Check if Ansible version is supported
-    debug: msg="Check Ansible version"
-    failed_when: ansible_version.full | version_compare('1.8', '<')
```

Comment:
Why do you want to remove the check? I think we only need to update `1.8` to `1.9', no?